contextIO.go

```diff
@@ -95,6 +100,23 @@ func (c *ContextIO) NewRequest(method, q string, queryParams url.Values, body io
 	return req, nil
 }
 
+func (c *ContextIO) wait() {
+	if c.rate > 0 && c.lastRequest.Unix() > 0 {
+		sinceLast := time.Now().Sub(c.lastRequest)
+		sleep := c.rate - sinceLast
+		if sleep > 0 {
+			time.Sleep(sleep)
+		}
+	}
+	c.lastRequest = time.Now()
+}
+
+// SetRate the number of possible requests per minute
+func (c *ContextIO) SetRate(requestsPerMinute float64) {
```

Comment:
Probably correct @AAAton.  Context.IO has the rate limit docs up now:  https://context.io/docs/lite/rate-limits and https://context.io/docs/2.0/rate-limits.  It would take a fair bit of work to implement them all.